app/controllers/users/edit_email_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Users
+  class EditEmailController < ApplicationController
+    before_action :confirm_two_factor_authenticated
+
+    def edit
+      @update_user_email_form = UpdateUserEmailForm.new(current_user)
+    end
+
+    def update
+      @update_user_email_form = UpdateUserEmailForm.new(current_user)
```

Comment:
Now that we have a 1-to-1 controller -> form mapping should we just call this `@form`?